lib/brain.js

```diff
@@ -49,6 +49,7 @@ const BigNumber = BN.klass
 
 let transitionTime
 
+const COINS_SUPPORTING_PAPER_WALLET = ['BTC', 'LTC', 'BCH']
```

Comment:
Instead of the hardcoded array here, consider replacing with the `getPaperWalletSupportiveCoins()` function we spoke about